pango/src/enums.rs

```diff
@@ -0,0 +1,147 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+#[cfg(any(feature = "v1_50", feature = "dox"))]
+#[cfg_attr(feature = "dox", doc(cfg(feature = "v1_50")))]
+use glib::{
+    error::ErrorDomain,
+    translate::*,
+    value::{FromValue, ToValue},
+    Quark, StaticType,
+};
+#[cfg(any(feature = "v1_50", feature = "dox"))]
+#[cfg_attr(feature = "dox", doc(cfg(feature = "v1_50")))]
+use std::fmt;
+
+#[cfg(any(feature = "v1_50", feature = "dox"))]
+#[cfg_attr(feature = "dox", doc(cfg(feature = "v1_50")))]
+#[derive(Debug, Eq, PartialEq, Ord, PartialOrd, Hash, Clone, Copy)]
+#[non_exhaustive]
+#[doc(alias = "PangoLayoutDeserializeError")]
+pub enum LayoutDeserializeError {
+    #[doc(alias = "PANGO_LAYOUT_DESERIALIZE_INVALID")]
+    Invalid,
+    #[doc(alias = "PANGO_LAYOUT_DESERIALIZE_INVALID_VALUE")]
+    InvalidValue,
+    #[doc(alias = "PANGO_LAYOUT_DESERIALIZE_MISSING_VALUE")]
+    MissingValue,
+    #[doc(hidden)]
+    __Unknown(i32),
+}
+
+#[cfg(any(feature = "v1_50", feature = "dox"))]
+#[cfg_attr(feature = "dox", doc(cfg(feature = "v1_50")))]
+impl LayoutDeserializeError {
+    #[doc(alias = "pango_layout_deserialize_error_quark")]
+    pub fn quark() -> Quark {
```

Comment:
Should implement https://gtk-rs.org/gtk-rs-core/git/docs/glib/error/trait.ErrorDomain.html instead

Reply:
You do a few lines below, but having a separate function for the quark is useless